.github/.wordlist.txt

```diff
@@ -757,6 +757,7 @@ js
 json
 JTAG
 Jupyter
+judgement
```

Comment:
The correct spelling in en-US is "judgment", for what it's worth, which is why you were getting a spellcheck issue...  Not sure how you want to handle that in this case.  Both spelling are used in en-GB, apparently, but with slightly different meanings/implications?

I'd suggest just using the en-US spelling and removing this wordlist entry.

Reply:
Sure, I will do that.